src/Schema/Schema.php

```diff
@@ -9,24 +9,23 @@
 use RalphJSmit\Laravel\SEO\Support\SEOData;
 use RalphJSmit\Laravel\SEO\Support\Tag;
 
+/**
+ * @deprecated Use CustomSchema paradigm
+ */
 abstract class Schema extends Tag
 {
     use Pipeable;
 
+    public string $tag = 'script';
+
     public array $attributes = [
         'type' => 'application/ld+json',
     ];
 
-    public string $context = 'https://schema.org/';
```

Comment:
This was not used

Reply:
I would say that it was included in the `get_object_vars()`?

Reply:
Actually it was not because the $attributes property defined above was overriding `get_object_vars`.
ps: More about that in your comment about `get_object_vars`

Reply:
Ok, good point 👍